simulation/tools/CMakeLists.txt

```diff
@@ -35,10 +41,10 @@ PCL_ADD_LIBRARY(${LIB_NAME} COMPONENT ${SUBSYS_NAME} SOURCES
   ${VTK_IO_TARGET_LINK_LIBRARIES}
   ${OPENNI_INCLUDES})
 target_link_libraries(${LIB_NAME} pcl_simulation pcl_common pcl_io
-  ${VTK_IO_TARGET_LINK_LIBRARIES} ${OPENGL_LIBRARIES} GLUT::GLUT)
+  ${VTK_IO_TARGET_LINK_LIBRARIES} ${OPENGL_LIBRARIES} ${_glut_target} GLEW::GLEW)
```

Comment:
Is GLEW::GLEW required now?

Reply:
Yeah. Compilation failed with missing glew symbols.